e2e/ansible/playbooks/hyperconverged/test-resize-jiva-volume/resize-volume.yml

```diff
@@ -0,0 +1,281 @@
+# STEPS INVOLUVED:
+#1) Deploy OpenEBS
+#2) Deploy percona
+#3) Fill data
+#4) Get the node where application is running
+#5) Unmount the file system
+#6) Logout the iscsi target
+#7) Get the id of the volume
+#8) Modfiy the volume size
+#9) Restart the replicas
+#10) Expanc the file system
+#)11) Mount the file system
+
+
+- hosts: localhost
+
+  vars_files:
+    - resize-volume-vars.yml
+
+  tasks:
+
+    - block:
+    
+        - name: Check whether maya-apiserver pod is running
+          shell: source ~/.profile; kubectl get pods --all-namespaces | grep maya-apiserver
+          args:
+            executable: /bin/bash
+          register: result
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+          until: "'Running' in result.stdout"
+          delay: 30
+          retries: 5
+
+        - name: Get $HOME of K8s master for kubernetes user
+          shell: source ~/.profile; echo $HOME
+          args:
+            executable: /bin/bash
+          register: result_kube_home
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}" 
+ 
+        - name: Copy the percona definition yaml to k8s master
+          copy:
+            src: "{{ percona_def }}"
+            dest: "{{ result_kube_home.stdout }}"
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+        - name: Getting the node name
+          shell: kubectl get nodes -o wide  | awk {'print $1'} | awk 'FNR == 4{print}'
+          args:
+            executable: /bin/bash
+          register: node_name
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+  
+        - name: Creating label for the node
+          shell: kubectl label nodes {{node_name.stdout}} app=percona
+          args:
+            executable: /bin/bash
+          register: label
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+          until: '"labeled" in label.stdout'
+          delay: 10
+          retries: 3
+
+        - name: Deploying application
+          shell: source ~/.profile; kubectl apply -f "{{ percona_def }}"
+          args:
+            executable: /bin/bash
+          register: percona_status
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+          until: "'deployment \"percona\" created' in percona_status.stdout"
+          delay: 30
+          retries: 5
+        
+        - name: Checking if the application is deployed successfully
+          shell: kubectl get pods --all-namespaces | grep percona
+          args:
+            executable: /bin/bash
+          register: app_status
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+          until: "'Running' in app_status.stdout"
+          delay: 30
+          retries: 15
+    
+        - name: Getting PV name
+          shell: kubectl get pv | grep openebs-percona | awk {'print $1'}
+          args:
+            executable: /bin/bash
+          register: pv
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
```

Comment:
Can you perform the step to replace the PVC name based on test name and get PV from there : something like - `kubectl get pvc pgdata-pgset-0 -o custom-columns=:spec.volumeName --no-headers`